dotnet6/cookiecutter-aws-sam-quickstart-sns-dotnet/{{cookiecutter.project_name}}/test/project.Tests/{{cookiecutter.project_name}}.Tests.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
@neville-lewis please rename the test folder 
from:
/test/project.Tests/
to:
/test/{{cookiecutter.project_name}}.Tests/